src/components/PressableWithSecondaryInteraction/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * This is a special Pressable that calls onSecondaryInteraction when LongPressed, or right-clicked.
+ */
+import _ from 'underscore';
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {Pressable} from 'react-native';
+
+const propTypes = {
+    onSecondaryInteraction: PropTypes.func.isRequired,
+    children: PropTypes.node.isRequired,
+};
+
+class PressableWithSecondaryInteraction extends Component {
+    constructor(props) {
+        super(props);
+        this.pressableRef = React.createRef();
```

Comment:
Last time I checked we are not using `React.createRef()`. I am generally cool with this since `createRef()` adds the `current` and it takes an extra second to think about which one you are dealing with.